src/utils.ts

```diff
@@ -17,13 +39,23 @@ export function getMockedEndpointKey(input: RequestInfo | RegExp): string {
   return cleanUrl.endsWith('/') ? cleanUrl.slice(0, -1) : cleanUrl;
 }
 
+export function getMockedEndpointKeyForHttp(request: HttpRequest): string {
+  const parsedUrl = new URL(getUrlForHttp(request));
```

Comment:
don't we lose the ability to mock endpoint by regex like we do in the regular getMockedEndpointKey?

Reply:
Solved